src/Artsy/Router/__tests__/RouterLink.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { MockRouter } from "DevTools"
+import { mount } from "enzyme"
+import { Link } from "found"
+import React from "react"
+import { RouterLink } from "../RouterLink"
+
+describe("RouterLink", () => {
+  it("uses the <Link> component if within a router context", async () => {
+    const wrapper = await mount(
+      <MockRouter
+        routes={[
+          {
+            path: "/",
+            Component: () => {
+              return <RouterLink to="/foo">Foo</RouterLink>
+            },
+          },
+        ]}
+      />
+    ).renderUntil(enzyme => {
+      try {
+        return enzyme.find(Link).length > 0
+      } catch {
+        // Guard against p == null, which is the first render pass
```

Comment:
What does the `p` refer to in this comment? `props`?

Reply:
oo i forgot to update this var; will update